env.js

```diff
@@ -113,6 +113,7 @@ function setStorage() {
   env.profile_collection = readENV('MONGO_PROFILE_COLLECTION', 'profile');
   env.devicestatus_collection = readENV('MONGO_DEVICESTATUS_COLLECTION', 'devicestatus');
   env.food_collection = readENV('MONGO_FOOD_COLLECTION', 'food');
+  env.azurepush_collection = readENV('MONGO_AZUREPUSH_COLLECTION', 'azurepush');
```

Comment:
It will be difficult to change in the future because we'll need a conversion.  I agree it should be generic and it's a much easier change at this stage.  I've got the change made and tested on my local branch, so I'm happy to sync it.